src/strToArray.js

```diff
@@ -0,0 +1,9 @@
+module.exports = function(str, delm) {
```

Comment:
if there is a token in the list that is just whitespace the resulting array will include an empty string.
i believe it's b/c the trim is happening after the filter

ex: ```strToArray('1,  ,3')``` will return ```['1','','3']```

additionally, you can one line it by doing something like this 

```'1,2,3'.split(',').map(x => x.trim()).filter(x => x)```